### ISSUE TITLE
error: Uncaught ReferenceError in bundle created with Deno 1.5.2

### ISSUE THREAD

**Issue by jveres** [denoland/deno#8314]
```sh
echo "import createGraph from \"https://dev.jspm.io/ngraph.graph\"; console.log(createGraph);" > test.ts
deno run test.ts
Check file:///Users/jveres/temp/test.ts
[Function: createGraph]

deno bundle test.ts test.bundle.js
Bundle file:///Users/jveres/temp/test.ts
Check file:///Users/jveres/temp/test.ts
Emit "test.bundle.js" (10.65KB)

deno run test.bundle.js
error: Uncaught ReferenceError: _npmNgraphDew is not defined
    var eventify = _npmNgraphDew();
                   ^
    at dew (file:///Users/jveres/temp/test.bundle.js:10:20)
    at file:///Users/jveres/temp/test.bundle.js:304:21
```

**Comment by alexanderGalushka** [comment]
this might be related https://github.com/denoland/deno/issues/8285

**Comment by jveres** [comment]
Solved with the recent Deno `1.5.4` release.
